
BugGrabberDB = {
	["limit"] = 100,
	["errors"] = {
		[1] = {
			["message"] = "BugSack\\BugSack.lua:391: attempt to call method `BugGeneratedByBugSack' (a nil value)\nBugSack\\BugSack.lua:391: in function `func'\nDewdrop-2.0\\Dewdrop-2.0.lua:468: in function <...erface\\AddOns\\FuBar\\libs\\Dewdrop-2.0\\Dewdrop-2.0.lua:421>\n",
			["time"] = "16:17",
			["session"] = 1,
		},
		[2] = {
			["message"] = "<string>:\"BugSack generated this fake error.\":1: `=' expected near `generated'\n<evaluated>: in function `RunScript'\nBugSack\\BugSack.lua:387: in function `func'\nDewdrop-2.0\\Dewdrop-2.0.lua:468: in function <...erface\\AddOns\\FuBar\\libs\\Dewdrop-2.0\\Dewdrop-2.0.lua:421>\n",
			["time"] = "16:17",
			["session"] = 1,
		},
		[3] = {
			["message"] = "BugSack\\BugSack.lua:391: attempt to call method `BugGeneratedByBugSack' (a nil value)\nBugSack\\BugSack.lua:391: in function `func'\nDewdrop-2.0\\Dewdrop-2.0.lua:468: in function <...erface\\AddOns\\FuBar\\libs\\Dewdrop-2.0\\Dewdrop-2.0.lua:421>\n",
			["time"] = "16:17",
			["session"] = 1,
		},
	},
	["save"] = true,
	["session"] = 1,
}
